code/modules/mob/living/carbon/human/species_types/apid.dm

```diff
@@ -4,9 +4,12 @@
 	id = SPECIES_APID
 	bodyflag = FLAG_APID
 	default_color = "FFE800"
-	species_traits = list(LIPS,NOEYESPRITES)
+	species_traits = list(LIPS,NOEYESPRITES,MUTCOLORS)
 	inherent_traits = list(TRAIT_BEEFRIEND)
 	inherent_biotypes = list(MOB_ORGANIC,MOB_HUMANOID,MOB_BUG)
+	mutant_bodyparts = list("apid_stripes","apid_antenna","apid_headstripes")
+	default_features = list("apid_stripes" = "thick","apid_headstripes" = "thick", "apid_antenna" = "curled")
+	hair_color = "fixedmutcolor"
```

Comment:
Not sure how I feel about the colour, looks really out of place in the image posted in the PR

Reply:
Yeah, my image doesn't really do it justice. I just picked the first option available rather than customizing.

I do overall see the addition of a color palette as necessary to the Apid Species, as literally every other roundstart species has a choice in color.

Reply:
Then maybe instead of picking from a color palette you pick from a set of colors like ethereals?

Reply:
yea i tried to code that... couldnt figure out the nightmare codespagetti that handles those good luck and have fun if you feel like tackling it

Reply:
> those good luck and have fun if you feel like tackling it. no one gave me the light of day in #coding so i gave up

sheesh man, the pessimism really kills the momentum. It'll be fine.

Reply:
> yea i tried to code that... couldnt figure out the nightmare codespagetti that handles those good luck and have fun if you feel like tackling it. no one gave me the light of day in #coding so i gave up

The code that I commented above is based on Ethereal code, so I guess I can explain how you'd do it with that.

Go [here](https://github.com/BeeStation/BeeStation-Hornet/blob/303d76f39558b07e0b4c656a08c0c1bf2a4caac3/code/modules/client/preferences.dm#L1580). This will show you the `color_ethereal` input. You will need to define a proc akin to this to make the color selection work. ~~I am rusty on how globals work here, so you may need to reload aglobals list or something.~~ You will need to go to the flavor lists and add a `color_list_apids` like the one for ethereals [here](https://github.com/BeeStation/BeeStation-Hornet/blob/78e921abd34eeb65535cc1633e98ee0561c1407c/code/_globalvars/lists/flavor_misc.dm#L53).  You'll probably want to cover [all the same bases](https://github.com/BeeStation/BeeStation-Hornet/search?q=color_list_ethereal) that ethereal color covers just in case.